content/doc/index.adoc

```diff
@@ -4,26 +4,45 @@ title: Jenkins Documentation
 section: doc
 ---
 
+== What is Jenkins?
+
 Jenkins is a self-contained, open source automation server which can be used to
 automate all sorts of tasks such as building, testing, and deploying software.
+
 Jenkins can be installed through native system packages, Docker, or even run
-standalone by any machine with the Java Runtime Environment installed.
+standalone by any machine with the Java Runtime Environment (JRE) installed.
 
 == Guided Tour
 
-This guided tour will use the "standalone" Jenkins distribution which requires
-Java 8. A system with more than 512MB of RAM is also recommended.
+This guided tour introduces you to the basics of using Jenkins and its main
+feature, Jenkins Pipeline. This tour uses the \'standalone' Jenkins distribution,
+which runs locally on your own machine.
+
+=== Prerequisites
+
+* Java 8 (either a JRE or Java Development Kit (JDK) is fine)
+* https://www.docker.com/[Docker] (navigate to the \'Get Docker' link at the top
+  of the page to access the Docker download that's suitable for your platform)
+* A machine with:
```

Comment:
I'd put hardware requirements first.  
This might be only my own preference, but my reasoning is If my machine doesn't meet the hardware requirements, it won't matter if I have Java 8 on it.

Reply:
That's a good point! I've reversed the order of hardware and software prerequisites, putting the hardware ones first.